tests/check/check_test.go

```diff
@@ -36,6 +36,26 @@ var tuples = []*openfgav1.TupleKey{
 	tuple.NewTupleKey("team:openfga", "member", "user:github|iaco@openfga"),
 }
 
+func TestMatrixMemory(t *testing.T) {
+	testRunTestMatrix(t, "memory")
+}
+
+func testRunTestMatrix(t *testing.T, engine string) {
+	t.Cleanup(func() {
+		goleak.VerifyNone(t)
+	})
+	cfg := config.MustDefaultConfig()
```

Comment:
Don't know if it will cause problem.  However, Ideally we will check for goLeak as well